runtime/src/tests.rs

```diff
@@ -352,3 +352,31 @@ mod pallets_interact {
         });
     }
 }
+
+/// This module tests the correct computation of rewards for validators.
+mod payout {
+    use pallet_staking::EraPayout;
+
+    use crate::{Balance, Runtime, CENTS};
+
+    use super::new_test_ext;
+
+    /// Test that validators receive a cumulative reward that mimics the current emission of
+    /// $ZEN in the PoW Horizen blockchain for miners, which is a coinbase of 6.25 Zen for
+    /// each block every 2.5 minutes.
+    #[test]
+    fn payout_is_same_as_pow_coinbase() {
+        new_test_ext().execute_with(|| {
+            const POW_BLOCK_TIME_MILLIS: u64 = 150 * 1000;
+            const POW_BLOCK_COINBASE: Balance = 625 * CENTS;
+            assert_eq!(
+                <Runtime as pallet_staking::Config>::EraPayout::era_payout(
+                    0,
+                    0,
+                    POW_BLOCK_TIME_MILLIS
+                ),
```

Comment:
With just this test you don't have any warrant about your implementation is proportional to the era length. What about add also:

```rust
assert_eq!(<Runtime as pallet_staking::Config>::EraPayout::era_payout(
                    0,
                    0,
                    POW_BLOCK_TIME_MILLIS/10
                ), (POW_BLOCK_COINBASE/10, 0)
);
```

Reply:
It makes sense, thanks!